lms/djangoapps/grades/models.py

```diff
@@ -73,8 +76,9 @@ def json_value(self):
         for block_dict in list_of_block_dicts:
             block_dict['locator'] = unicode(block_dict['locator'])  # BlockUsageLocator is not json-serializable
         data = {
-            'course_key': unicode(self.course_key),
-            'blocks': list_of_block_dicts,
+            u'course_key': unicode(self.course_key),
+            u'blocks': list_of_block_dicts,
+            u'version': self.version,
```

Comment:
Very picky nit: put `version` as the first field since typically one expects parsing the version first in any protocol.